src/olympia/devhub/forms.py

```diff
@@ -593,17 +596,26 @@ class DescribeForm(AkismetSpamCheckFormMixin, AddonFormBase):
         widget=TransTextarea(), required=False,
         label=_(u'Please specify your add-on\'s Privacy Policy:'))
 
-    fields_to_akismet_comment_check = ['name', 'summary']
+    fields_to_akismet_comment_check = ['name', 'summary', 'description']
 
     class Meta:
         model = Addon
-        fields = ('name', 'slug', 'summary', 'is_experimental', 'support_url',
-                  'support_email', 'privacy_policy', 'requires_payment')
+        fields = ('name', 'slug', 'summary', 'description', 'is_experimental',
+                  'support_url', 'support_email', 'privacy_policy',
+                  'requires_payment')
 
     def __init__(self, *args, **kw):
         kw['initial'] = {
             'has_priv': self._has_field('privacy_policy', kw['instance'])}
         super(DescribeForm, self).__init__(*args, **kw)
+        content_waffle = waffle.switch_is_active('content-optimization')
```

Comment:
That waffle switch sounds very broad, do we plan to put more functionality behind it?

Reply:
yes, it's all (well, the requirement changes are) going to be enabled end of November at once https://github.com/orgs/mozilla/projects/39#card-13476230

Reply:
gotcha, thanks